lib/daru/vector.rb

```diff
@@ -884,40 +884,29 @@ def to_s
     end
 
     # Create a summary of the Vector using Report Builder.
-    def summary(method=:to_text)
-      ReportBuilder.new(no_title: true).add(self).send(method)
-    end
-
-    # :nocov:
-    def report_building b # rubocop:disable Metrics/AbcSize,Metrics/MethodLength
-      b.section(name: name) do |s|
-        s.text "n :#{size}"
-        s.text "n valid:#{count_values(*Daru::MISSING_VALUES)}"
-        if @type == :object
-          s.text  "factors: #{factors.to_a.join(',')}"
-          s.text  "mode: #{mode}"
-
-          s.table(name: 'Distribution') do |t|
```

Comment:
Well then I will try to make a custom implementation but i do not understand what this table actually does . I checked with some example and the thing is `frequencies.sort_by(&:to_s).each do |k,v|` returns a vector , so why use two variable k,v. Please help me in this